java/connector-node/risingwave-sink-jdbc/src/main/java/com/risingwave/connector/JDBCSink.java

```diff
@@ -244,6 +249,34 @@ private PreparedStatement generatePreparedStatement(
                 case BYTEA:
                     stmt.setBinaryStream(placeholderIdx++, (InputStream) row.get(i));
                     break;
+                case LIST:
+                    var val = row.get(i);
+                    // JSON payload returns a List, but JDBC expects an array
+                    if (val instanceof java.util.List<?>) {
+                        val = ((java.util.List<?>) val).toArray();
+                    }
+                    assert (val instanceof Object[]);
+                    Object[] objArray = (Object[]) val;
+                    if (targetDbType == DatabaseType.POSTGRES) {
+                        var fieldType = column.getDataType().getFieldType(0);
+                        var sqlType =
+                                JdbcUtils.getDbSqlType(
+                                        targetDbType, fieldType.getTypeName(), fieldType);
+                        stmt.setArray(i + 1, conn.createArrayOf(sqlType, objArray));
+                    } else {
+                        // convert Array type to a string for other database
+                        // reference:
+                        // https://dev.mysql.com/doc/workbench/en/wb-migration-database-postgresql-typemapping.html
+                        StringBuilder sb = new StringBuilder();
+                        for (int j = 0; j < objArray.length; j++) {
+                            sb.append('"').append(objArray[j].toString()).append('"');
```

Comment:
1. SQL Databases usually use single quotes instead of double quotes.
2. I think we should not add the quotes for numbers (int/float/decimal)?
3. Pay attention to the string escape (`"` -> `\"` or `""`, need to look the docs)